src/phase_find.c

```diff
@@ -161,12 +163,14 @@ static ini_option_t option[] = {
   {"external-mpi-args", "Startup arguments for external scripts, some MPI's may not support this!", 0, INI_STRING, "", & of.ext_mpi},
   {"external-extra-args", "Extra arguments for the external scripts", 0, INI_STRING, "", & of.ext_args},
   {"nproc", "Set the number of processes for pfind/the external script", 0, INI_UINT, NULL, & of.nproc},
+  {"noRun", "Disable running of this phase", 0, INI_BOOL, NULL, & of.no_run},
```

Comment:
I probably should have made this patch on a separate branch, since it again doesn't have anything to do with the non-POSIX backends.